apps/system/js/remote/app.js

```diff
@@ -0,0 +1,31 @@
+/* global BaseModule */
+'use strict';
+
+(function(exports) {
+  var App = function() {
+  };
+
+  App.SUB_MODULES = [
+    'WallpaperManager',
+    'remote/SimpleAppWindowManager'
+  ];
+
+  App.EVENTS = [
+    'wallpaperchange'
+  ];
+
+  BaseModule.create(App, {
+    DEBUG: false,
+    name: 'App',
+    _start: function() {
+      if (window.location.hash) {
+        this.displayId = parseInt(window.location.hash.substring(1), 10);
```

Comment:
There may be more than one external display at the same time so we need to distinguish them in BroadcastChannel by using `displayId` which is passed from gecko.